mitosheet/src/components/footer/DataSheetTabActions.tsx

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Mito
+
+import React, { useEffect } from 'react';
+import MitoAPI, { getRandomId } from '../../api';
+import { GraphDataJSON, GraphID, UIState } from '../../types';
+import Dropdown from '../elements/Dropdown';
+import DropdownItem from '../elements/DropdownItem';
+import { ModalEnum } from '../modals/modals';
+import { TaskpaneType } from '../taskpanes/taskpanes';
+
+/*
+    Helper function for finding all of the graph tab names
+    that are created from a given sheet index
+*/
+export const getGraphTabNamesAndIDsFromSheetIndex = (sheetIndex: number, graphDataJSON: GraphDataJSON): 
```

Comment:
Btw: This type of comment isn't generally super helpful. most of the time, I just did what I thought was best so if you could be more specific about what you want, that would be a more helpful comment. Happy to change to whatever you want, just have to know what that is. 

In this case, I think it is properly formatted because that is how VSCode told me to format is. But I'll update it to all being in one line.